configure.ac

```diff
@@ -136,7 +136,7 @@ dnl AC_FUNC_REALLOC
 AC_TYPE_SIGNAL
 AC_FUNC_STAT
 AC_FUNC_STRFTIME
-AC_CHECK_FUNCS([issetugid inet_pton])
+AC_CHECK_FUNCS([issetugid])
```

Comment:
I'm updating the package for lighttpd 1.4.40 in Haiku.
The nightly builds of Haiku are available for 3 architectures: x86_gcc2, x86 and x86_64.
x86 and x86_64 use recent versions of gcc. But x86_gcc2 uses a slightly modified version of gcc-2.95.3, which means it does not accept most of what newer versions of gcc accept.

https://github.com/fbrosson/haikuports/tree/lighttpd/www-servers/lighttpd/patches
has the patches we need for lighttpd-1.4.40 to make it build on Haiku/x86_gcc2.
Do you think any of these two patches:
* [lighttpd-1.4.40-gcc2-mod_ssi-and-test_base64.patchset](https://github.com/fbrosson/haikuports/blob/lighttpd/www-servers/lighttpd/patches/lighttpd-1.4.40-gcc2-mod_ssi-and-test_base64.patchset)
* [lighttpd-1.4.40-gcc2-vector_config_weak.patchset](https://github.com/fbrosson/haikuports/blob/lighttpd/www-servers/lighttpd/patches/lighttpd-1.4.40-gcc2-vector_config_weak.patchset)

could be accepted in lighttpd?
Please feel free to answer no, since we are already used to having plenty of gcc2 patches for many programs ;-)
BTW, I'll dig why I added a **`(unsigned char)`** cast in some chunks but not all.
Thanks!